### PR TITLE
add logging configs for algo host

### PR DESCRIPTION
**Pull request by avsek477**
- Adds logging directory and option for algo host
- Pass logging options to algo host

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ const HttpProxy = require('./lib/bfx_api_proxy')
 const syncMarkets = require('./lib/sync_meta')
 const capture = require('./lib/capture')
 
-const { algos } = require('./config/algo_server.conf.json')
+const { algos, logAlgo = false, logAlgoDir = '' } = require('./config/algo_server.conf.json')
```
[the remark this answers is not in the document]
Comment (avsek477) [reply index.js:15]: thanks!! will make the change

**Review by vitaliystoliarovcc — APPROVED** [review]
lgtm

File: index.js [hunk 2]
```diff
@@ -32,14 +33,20 @@ module.exports = async ({
     adapter: HFDBLowDBAdapter({ dbPath: algoDBPath })
   })
 
+  const logAlgoOpts = {
+    logAlgo: ALGO_LOG ? ALGO_LOG === 'true' : false,
+    logAlgoDir: ALGO_LOG_DIR || ''
```
Comment (robertkowalski) [inline index.js:38]: how about using a default here that points to `~/.honeyframework/algo-logs` or similar?

Comment (robertkowalski) [reply index.js:38]: https://github.com/bitfinexcom/bfx-hf-ui/blob/985b31931a3aa742ee018507c40d857cad537382/scripts/start-api-server.js#L15

Comment (robertkowalski) [reply index.js:38]: nevermind, lets add this with the HF UI setting for algo logs

File: index.js [hunk 3]
```diff
@@ -32,14 +33,20 @@ module.exports = async ({
     adapter: HFDBLowDBAdapter({ dbPath: algoDBPath })
   })
 
+  const logAlgoOpts = {
+    logAlgo: ALGO_LOG ? ALGO_LOG === 'true' : false,
```
Comment (robertkowalski) [inline index.js:37]: won't work with the example, where a boolean is passed

**Review by robertkowalski — APPROVED** [review]
lgtm

**Pull request merged**